tests/test_structure.py

```diff
@@ -2,67 +2,53 @@
 
 import os
 
+import shutil
+
 from wrapper_writer.structure import Structure
 
 class TestStructure(TestCase):
     path = "./tests/dir/tests"
     template = ""
     file_name_format = ""
-
-    @classmethod
-    def delete_directories(cls, path, absolute_path):
-        directories = path.split("\\")
-        for i in range(len(directories), 1, -1):
-            latest_path = "\\".join(directories[0:i])
-            os.rmdir(os.path.join(absolute_path, latest_path))
-
-    @classmethod
-    def tearDownClass(cls):
-        absolute_path = os.getcwd()
-
-        directories = cls.path.split("\\")
-        for i in range(len(directories), 1, -1):
-            latest_path = "\\".join(directories[0:i])
-            os.rmdir(os.path.join(absolute_path, latest_path))
-
-        directories = "./tests/already/here".split("\\")
-        for i in range(len(directories), 1, -1):
-            latest_path = "\\".join(directories[0:i])
-            os.rmdir(os.path.join(absolute_path, latest_path))
-
+    project_root = ""
 
 
     def test_create_path(self):
         absolute_path = os.path.join(os.getcwd(), self.path)
 
         # Calling the structure class
-        s = Structure(self.path, self.template, self.file_name_format)
+        s = Structure(self.project_root, self.path, self.template, self.file_name_format)
+        s.project_root = os.getcwd()
         s.create_path()
         full_path = s.full_path
 
-        self.assertEquals(absolute_path, full_path)
+        self.assertEqual(absolute_path, full_path)
 
     def test_create_dir(self):
         absolute_path = os.path.join(os.getcwd(), self.path)
 
         # Calling the structure class
-        s = Structure(self.path, self.template, self.file_name_format)
+        s = Structure(self.project_root, self.path, self.template, self.file_name_format)
         s.full_path = absolute_path
-        s.create_dir()
 
+        s.create_dir()
         self.assertTrue(os.path.isdir(absolute_path))
+        shutil.rmtree("./tests/"+self.path.split("/")[2])
```

Comment:
If this test errors it wont remove the directory

Reply:
done